config/apidoc.php

```diff
@@ -14,6 +14,14 @@
      */
     'router' => 'laravel',
 
+    /*
+     * URL prefix for Laravel routes.
+     */
+
+    'url_prefix' => '/doc',
+
+    'middleware' => [],
```

Comment:
Would you like to doc-block this?

Reply:
Right! Forgot to add a description. Soon I will be at the computer and can do it.